sdcm/cluster.py

```diff
@@ -1802,6 +1811,7 @@ class BaseScyllaCluster(object):
 
     def __init__(self, *args, **kwargs):
         self.termination_event = threading.Event()
+        self.nemesis_lock = threading.Lock()
```

Comment:
see comment below, locking should be done on BaseNode

Reply:
done